pcsx2/IopMem.cpp

```diff
@@ -96,6 +96,10 @@ void iopMemoryReserve::Reset()
 		psxMemWLUT[i + 0x2000 + 0x1e00] = (uptr)&eeMem->ROM1[i << 16];
 	}
 
+	for (int i = 0; i < 0x0008; i++) {
+        psxMemWLUT[i + 0x2000 + 0x1e40] = (uptr)&eeMem->ROM2[i << 16];
+    }
```

Comment:
Would be nice to fix the formatting throughout the pr. Files use tabs for indentation.

Reply:
Should be ok now for all files.